pkgs/top-level/python-packages.nix

```
@@ -4980,6 +4980,8 @@ in {
   python-simple-hipchat = callPackage ../development/python-modules/python-simple-hipchat {};
   python_simple_hipchat = self.python-simple-hipchat;
 
+  python-scs = callPackage ../development/python-modules/python-scs { };
```

this should try to match what is published on pypi, if this is https://pypi.org/project/scs/ it should be
```suggestion
  scs = callPackage ../development/python-modules/python-scs { scs = pkgs.scs; };
```

>  One of the arguments is the name of the package, which consists of a basename (generally following the name on PyPi) and a version. 

from: https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#packaging-a-library

---

also, the directory name should change.

I know this kind of contradicts what @bhipple said, but the python-module name should align with upstream intentions.

---

if a package requires scs, then declaring it as a dependency in their package.nix should be very similar to their requirements.txt or setup.py